intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -185,9 +190,12 @@ def log_message(self, msg):
         # Read msg-body and add as raw to a new report.
         # now it's up to a parser to do the interpretation of the message.
         if raw_msg:
-            report = self.new_report()
-            report.add("raw", raw_msg)
-            self.send_message(report)
+            try:
+                report = self.new_report()
+                report.add("raw", raw_msg)
+                self.send_message(report)
+            except:
```

Comment:
Did you see a case where this happened? Which Error is thrown? Please specify it.